packages/playground/website/src/components/error-report-modal/index.tsx

```diff
@@ -0,0 +1,101 @@
+import { useEffect, useState } from 'react';
+import Modal from '../modal';
+import { addFatalErrorListener, logger } from '@php-wasm/logger';
+import { Button, TextareaControl, TextControl } from '@wordpress/components';
+
+import css from './style.module.css';
+
+export function ErrorReportModal() {
+	const [hasError, setHasError] = useState(false);
+	const [text, setText] = useState('');
+	const [logs, setLogs] = useState('');
+	const [url, setUrl] = useState('');
+	const [submitted, setSubmitted] = useState(false);
+
+	useEffect(() => {
+		addFatalErrorListener(logger, (e) => {
+			setHasError(true);
+			setText('');
+			setLogs((e as CustomEvent).detail.logs.join(''));
+			setUrl(window.location.href);
+		});
+	}, []);
+
+	function onClose() {
+		setHasError(false);
+		setText('');
+		setLogs('');
+		setUrl('');
+		setSubmitted(false);
+	}
+
+	function onSubmit() {
+		const data = ['What happened?', text, 'Logs', logs, 'Url', url].join(
+			'\n\n'
+		);
+		console.log(data);
+		// TODO: send data to the server
+		setSubmitted(true);
+	}
+
+	return (
+		<Modal isOpen={hasError} onRequestClose={onClose}>
+			{submitted && (
+				<header className={css.errorReportModalHeader}>
+					<h2>Thank you for reporting the error</h2>
+					<p>
+						We will look into the issue and and open an{' '}
```

Comment:
```suggestion
						We will look into the issue and open an{' '}
```

Also, just to make sure, the error reports will go to Slack and we're saying here that we may manually turn them into a new issue on GitHub, right? But then we also link to GitHub issues. Could this instead communicate the error report was sent to Slack and provide a link to the message? It would also be nice to communicate the next steps and why we're not creating a GitHub issue from the get go.